### ISSUE TITLE
The name is not correctly defined for "show text label" button of the Dashboard. : A11y_SQL Azure Data Studio_Dashboard_AI4D

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/azuredatastudio#26397]
### Tags:
#A11yTCS;#A11yWCAG;#WCAG4.1.2;#Name Role Value;#A11ySev2;#Win32;#DesktopApp;#BM_SQL Azure Data Studio_Win32_Sep25;#SQL Azure Data Studio;
**"[[Check out Accessibility Insights!]](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**
### Environment Details:
Application Name: SQL Azure Data Studio
Version: 1.52.0 (system setup)
VS Code: 1.82.0
Electron: 30.5.1
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22621
### Repro Steps:
1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server connection. 
4. Then select 'Manage' from the context menu to view the dashboard.
5. Tab till show/hide panel button and inspect it.
7. Observe that The name is not correctly defined for "show text label" button of the Dashboard. 
### Actual:
The name is not correctly defined for "show text label" button of the Dashboard. The purpose and functionality is not matching with the name defined for the "show text label" button of Dashboard.

### Expected:
The name should be defined correctly which can determine the purpose of the button, here the name should be defined as show/hide panel instead of "show text label" button.

### User Impact:
Keyboard users will not be able to get the additional information provided about the fields as tooltips are not accessed with keyboard.
### Recommendations:
 [MAS Problems and Suggested Fixes](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fmicrosoft.sharepoint.com%2fsites%2faccessibility%2fLists%2fMASProblemsAndSuggestedFixes%2fReviewed%2520Items.aspx&data=04%7c01%7cv-prajadhav%40microsoft.com%7cf642cb2fc53d4c4be38408d9d97cf216%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637779952946112692%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000&sdata=avswAOFlXEiWy%2fRT%2f%2fPGnurQzMGOUdSW7Z5Xp8ZioiQ%3D&reserved=0)
### MAS Reference:

- [MAS 4.1.2: Name, Role, Value Guide](https://microsoft.sharepoint.com/sites/accessibility/SitePages/MAS-4.1.2-Name-Role-Value-Guide.aspx)
- [MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)
